lib/index.js

```diff
@@ -16,46 +16,42 @@ require('bedrock-express');
 
 var brPassport = require('bedrock-passport');
 var ensureAuthenticated = brPassport.ensureAuthenticated;
-var optionallyAuthenticated = brPassport.optionallyAuthenticated;
 
 require('./config');
 
 // TODO: Add logging info
 var logger = bedrock.loggers.get('app');
 
 bedrock.events.on('bedrock-express.configure.routes', function(app) {
-  var basePath = config['messages-client-http'].basePath;
+  var basePath = config['messages-client-http'].routes.basePath;
   // Get ALL clients.
   app.get(basePath, ensureAuthenticated, function(req, res, next) {
-      brMessagesClient.getAll(req.user.identity, {}, function(err, clients) {
-        if(err) {
-          return next(err);
-        }
-        res.json(clients);
-      });
+    brMessagesClient.getAll(req.user.identity, {}, function(err, clients) {
+      if(err) {
+        return next(err);
+      }
+      res.json(clients);
     });
+  });
 
-  // The restart-job endpoint is operating on the premise that there can only
+  // TODO: Might want to model this as a PATCH update if we aren't supporting
+  // adding message clients via post.
```

Comment:
Even if we aren't adding them via POST, I think we should model it that way. The other way has an expectation of creating something because all of the other APIs work that way. Add an issue to fix it up later -- we'll have to add some backwards compatibility support for this old method.